src/main/java/com/webank/wecross/routine/xa/XATransactionManager.java

```diff
@@ -761,8 +806,15 @@ public void asyncListXATransactions(
                 }
             }
 
-            ListXAReduceCallback reduceCallback =
-                    getListXAReduceCallback(offsets.size(), offsets, size, callback);
+            ListXAReduceCallback reduceCallback = null;
```

Comment:
Using whether chainPath is passed in or not as a compatibility judgement, it is unnecessary to use version.